.github/workflows/workflow.yaml

```diff
@@ -40,6 +40,24 @@ on:
       trivy-error-is-success:
         type: boolean
         default: false
+      terraform-docs-job-enabled:
+        type: boolean
+        default: true
```

Comment:
Jeg vet ikke om det er så smart med default `true`. Det er jo praktisk for oss nå, med så mange modul-repoer, men de config-repoene som bruker dette vil vel ikke ha dette?

Reply:
Du får ta veto her. Jeg er helt i limbo mellom true eller false for vårt bruk her nå

Reply:
egentlig, så hadde det vært nice med doc på config-repoene også.
Det er ikke så lett å få oversikt over alle moduler som brukes i et config-repo. og docs action vil jo nøste opp i alle .tf filer
might be usefull?

Reply:
Det har jeg ikke tenkt på. Vi prøver da :)